quickstarts/synthetics/private-locations/config.yml

```diff
@@ -0,0 +1,58 @@
+id: 
```

Comment:
Hey @keegoid-nr  - We generate the ID automatically here after merging in a new quickstart:
https://github.com/newrelic/newrelic-quickstarts/blob/ed53333e26e9d8964fd3e245459f1c91d60fc0f3/utils/generate-uuids.js#L15-L20

We just need to remove this line from the config file. Other than that, everything looks great!